src/app/components/site/cs/HomepageCS.tsx

```diff
@@ -1,233 +1,201 @@
-                            image: {src: "/assets/cs/decor/lella.png"},
-                            bodyText: "Read our interviews with computer scientists who are doing amazing things in a huge range of computing-related fields.",
-                            clickUrl: "/pages/computer_science_stories",
-                            buttonText: "Read more",
-                            className: "bg-cultured-grey",
-                        }}/>
-                        <AdaCard card={{
-                            title: "Online professional development",
-                            image: {src: "/assets/cs/decor/teacher-2.png"},
-                            bodyText: "20+ training courses, written specifically for computing teachers. No programming experience is needed. Great for keeping up with CPD requirements.",
-                            clickUrl: "/pages/online_courses",
-                            buttonText: "Find out more",
-                            className: "bg-cultured-grey",
-                        }}/>
+                    <ColumnSlice className={"align-items-start"}>
+                        <TextBlock className="backslash-left text-white">
+                            <h2 className={"font-size-1-75"}>
+                                &rdquo;Ada Computer Science has eliminated the need for textbooks for A level computer science. There is rarely a need for any other sources of information when planning lessons and it’s free!&rdquo;
+                            </h2>
+                            <p>– Computer science teacher</p>
+                        </TextBlock>
+                        <TextBlock className="backslash-left text-white">
+                            <h2 className={"font-size-1-75"}>
+                                &ldquo;
+                                    I love Ada CS! The content featured is very comprehensive and detailed, and the visual guides through topics like sorts are particularly helpful to aid my understanding.
+                                &rdquo;
+                            </h2>
+                            <p>– Computer science student</p>
+                        </TextBlock>
                     </ColumnSlice>
                 </Container>
             </section>
 
             <section id="what-resources">
                 <Container className={"homepage-padding mw-1600"}>
                     <TextBlock md={8}>
-                        <h2 className={"font-size-1-75 mb-4"}>Teaching outside the UK?</h2>
-                        <p>We&apos;ve organised our learning resources by prior knowledge and age group to make them easy to adapt for curricula around the world.</p>
+                        <h2 className={"font-size-1-75 mb-4"}>Exam board alignment</h2>
+                        <p>We&apos;ve organised our learning resources so they can be easily used, wherever you are.</p>
                     </TextBlock>
                     <ColumnSlice>
```

Comment:
This changes layout at `md` in the designs:
```suggestion
                    <ColumnSlice breakpoint="md">
```